### DISCUSSION TITLE
[Feature] Deprecated warnings on npmjs.com

### DISCUSSION THREAD

**Discussion by ex-tag** [npm/feedback#744] [Website]
Everytime I run `npm install` I see deprecated warnings for packages. I think these warnings should also be listed on NPM (www.npmjs.com), so that developers can access if the package has outdated dependencies before they install. Furthermore, I think that NPM should send an email to the owner of the package, with the warnings... if that's not already being done.

**Comment by darcyclarke** [comment]
Hey! I think this kind of feedback is better left in the generic Feedback Discussions here: https://github.com/npm/feedback/discussions There's a category there specific to "website". This repo & discussions are specific to the CLI.
